### DISCUSSION TITLE
Duda parte 1 acerca de la cápsula

### DISCUSSION THREAD

**Discussion by gaspar-mr05** [IIC2133-PUC/2024-1#82] [Tarea 1] [answered]
Hola! Estaba viendo la cápsula de la parte 1 y veo que lo hacen con heaps. Yo sin embargo cumpliendo lo que pedía el enunciado sobre la complejaidad decidi ocupar un árbol avl y en verdad me sirvió mucho, me corrieron todos los test y todo bien, pudiendo argumentar porque esa estructura es válida y cumple la complejidad. Es válido ocupar avl en la parte 1? o obligatoriamente había que hacerlo con heap?

**Comment by sebacarrascop** [comment, marked as answer]
hola, mientras lo argumentes bien está bien!! saludos.
